docs/_static/css/custom.css

```diff
@@ -29,6 +29,7 @@ iframe.demo {
   height: 250px;
   border: 1px solid var(--color-background-border);
   border-radius: 5px;
+  padding-top: 1rem;
```

Comment:
padding came form the `components.py` actually. it was modified there from an old version of marimo which had the large padding on it